source/session.ts

```diff
@@ -666,28 +668,30 @@ export class Session {
    *   Return update or create promise.
    */
 
-  ensure<T extends Data = Data>(
```

Comment:
I'd rather take this separately if we can't do it properly right now. Don't like creating a `const` just to cast the data as `any`.

Reply:
Alright, noted. Will correct this soon 👍

Reply:
I have corrected it! 😄